recipes/Remi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "remi" %}
+{% set version = "2020.1.16" %}
+{% set styledName = "Remi" %}
```

Comment:
Also, you don't use `styledName`anywhere, so you can just remove that line from the recipe.